mu-plugins/blocks/query-filter/render.php

```diff
@@ -146,15 +155,18 @@ class="wporg-query-filter__modal-close"
 			?>
 
 			<div class="wporg-query-filter__modal-actions">
+				<?php if ( $has_multiple ) : ?>
```

Comment:
This throws a JS exception since the clear button is conditional:
https://github.com/WordPress/wporg-mu-plugins/blob/21c8d937b1c6003d5fb5a2598278fd1d7905a547/mu-plugins/blocks/query-filter/src/view.js#L73

Reply:
Checking for `clearButton` now, in https://github.com/WordPress/wporg-mu-plugins/pull/596/commits/dcd4af908577d1250487c8bcd93c2115d651e78b